src/components/variantsListPanel/VariantsPanelList.vue

```diff
@@ -62,19 +61,9 @@ export default {
     },
     methods: {
         variantByUrl() {
-            const urlAPI = require('url');
-            const parameters = window.location.search.slice(1).split('&');
-            const variant = parameters.find(item => {
-                const split = item.split('=');
-                if (split) {
-                    return split[0].trim() === 'variant';
-                }
-            });
-            if (variant && (variant.split('=').length > 1)) {
-                const id = Number(variant.split('=')[1]);
-                if (id && id >= 1 && id <= this.data.length) {
-                    this.selectItem(id-1);
-                }
+            const id = Number(this.$route.query.variant);
```

Comment:
I guess the idea was that url should be always up to date, for example if you change variant url should be changed as well.
Besides try to select first variant, it's still not possible to choose it.
I suppose it's similar feature to workspace selection.